v2/test/utils/helpers.go

```diff
@@ -0,0 +1,77 @@
+package utils
```

Comment:
While this package and its location might look odd I have much more tests coming in the follow-up PRs. Utils are reused between examples and tests.